ports/nrf/examples/nrf9160_gps.py

```diff
@@ -0,0 +1,46 @@
+# GNSS sample for pca10090
+
+import network
+import socket
+
+n = network.NRF91()
+
+at = socket.socket(socket.AF_LTE, socket.SOCK_DGRAM, socket.PROTO_AT)
```

Comment:
Is it the right pattern to have a socket for the AT connection/commands?  Wouldn't it be simpler to have the method on the `n` cellular network object, eg `n.at_send(...)` and `n.at_recv(...)`?  Again I'm thinking about other ports that may have cellular, even as an external component.

Reply:
Yeah, this is the pattern to create a AT-socket. Directly dispatched to the BSD-library. I did actually add a convenience function similar to what you suggest, modnwlte.c:828.

It's a one shot though, which only returns the response, then closes the at-socket again. But it should not be a problem to keep this open for a longer time in case you also want to do subscriptions. One reason for doing a socket instead of a convenience function like this is that you might want to also add it to a socket `poll()` mask.